### DISCUSSION TITLE
featureCounts Refseq Fix for bacteria

### DISCUSSION THREAD

**Discussion by Groshack** [biostars/biostar-handbook-issues#211] [General]
Hi everyone, 

Thank for so much for writing RNA-Seq by Example. I am new to all this and this book has been a life saver. I just want to post my fix to Subread's FeatureCounts when using RefSeq.gtf files. In my situation FeatureCounts was finding 76/1996 features of my prokaryotic organism. 

I amended the code to find the feature 'gene' and it worked. 

Original line: 
featureCounts -p -a refs/features.gff -o counts.txt bam/BORED_?.bam bam/EXCITED_?.bam
 
featureCounts -p -F GTF -t 'gene' -a refs/refseq.gtf -o counts.txt bam/1_?.bam bam/2_?.bam

I hope this finds someone in need! 

Cheers,
Groshack

**Comment by ialbert** [comment]
Thanks for the note, I will merge this recommendation into the book in the near future

**Reply by Groshack** [reply]
You're a champion!
